engine/src/main/java/org/entando/entando/aps/system/services/component/ComponentUsageDetails.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.aps.system.services.component;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+import lombok.Setter;
+
+@JsonPropertyOrder({"type", "code", "exist", "usage", "status", "extraProperties", "references"})
+public class ComponentUsageDetails extends ComponentUsage implements Serializable {
```

Comment:
I created this class because I didn't want to alter the super classes; these classes are used in different parts of the system and I didn't want to risk changing the behavior of the services that use it. Changing super classes only affects the package name